src/bandersnatch_filter_plugins/latest_name.py

```diff
@@ -16,6 +16,7 @@ class LatestReleaseFilter(FilterReleasePlugin):
 
     name = "latest_release"
     keep = 0  # by default, keep 'em all
+    latest: Sequence = []
```

Comment:
Do we know what's always going to be in the sequence? If so lets add it - e.g. Sequence[str]